classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -683,7 +683,7 @@ protected function display_filter_by_status() {
 			}
 
 			if ( $status_name === $request_status || ( empty( $request_status ) && 'all' === $status_name ) ) {
-				$status_list_item = '<li class="%1$s"><strong>%3$s</strong> (%4$d)</li>';
+				$status_list_item = '<li class="%1$s"><a href="%2$s"><strong>%3$s</strong></a> (%4$d)</li>';
```

Comment:
```suggestion
				$status_list_item = '<li class="%1$s"><a href="%2$s" class="current">%3$s</a> (%4$d)</li>';
```

↑ I think this will give us better consistency with other list tables.